src/terminal/Terminal.gd

```diff
@@ -0,0 +1,42 @@
+class_name Terminal
+extends Node
+
+@export var start_window : BaseTerminalWindow;
+
+@onready  var content : MarginContainer = $ContentContainer;
+
+var window_stack : Array;
+
```

Comment:
this array should be typed, unless you explicitly want to use a variant array

Reply:
i'd be fine with this type being `Node` if you didn't refer to `visible` later on without any type safety checks. Changing this to `CanvasItem`.